swift-pipeline

```diff
@@ -99,6 +100,27 @@ parser.add_argument(
     ),
 )
 
+parser.add_argument(
```

Comment:
This makes me quite uncomfortable as this is something that I see should be handled through the stylesheet. We could add a command line argument to override the stylesheet in the config file, I suppose?